docs/docs/sql-syntax/functions/list-map/splice.md

```diff
@@ -0,0 +1,63 @@
+# SQL Function - "SPLICE"
+
+The "SPLICE" function in GlueSQL is used to modify elements in a list. It allows you to remove or replace elements in a list. The syntax for the "SPLICE" function is as follows:
+
+```sql
+SPLICE(list_expression, start, end [, replace_elements])
+```
+
+- `list_expression`: The list you want to modify.
+- `start`: The position at which you want to start the modification.
+- `end`: The exclusive end position for the modification.
+- `replace_elements` (optional): A list of elements to insert in place of the removed elements.
+
+## Example
+
+Let's consider a table named "ListTable" with the following data:
+
+```sql
+CREATE TABLE ListTable (
+    id INTEGER,
+    items LIST
+);
+
+INSERT INTO ListTable VALUES
+    (1, '[1, 2, 3]'),
+    (2, '["1", "2", "3"]'),
+    (3, '["1", 2, 3]');
+```
+
+Now, we can use the "SPLICE" function to modify the list in various ways:
+
+1. Remove elements from a list:
+
+```sql
+-- Remove elements at positions 1 to 3 from the list '[1, 2, 3]'
+SELECT SPLICE(CAST('[1, 2, 3, 4, 5]' AS List), 1, 4) AS result;
+-- Output: '[1, 4, 5]'
```

Comment:
it looks.. it should be `1, 3`, not `1, 4`

Reply:
Thank you!